src/database/models/accomodation.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+/////////////////////////////////
+
+'use strict';
+import { Model } from 'sequelize';
+export default (sequelize, DataTypes) => {
+  class accomodation extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate({Room}) {
+     this.hasMany(Room,{foreignKey:"accomodationId",as:"rooms"})
+    }
+  }
+  accomodation.init({
+    name: DataTypes.STRING,
+    description: DataTypes.STRING,
+    location: DataTypes.STRING,
+    image: DataTypes.STRING,
+    availability: DataTypes.STRING,
+    highlight: DataTypes.STRING,
+    amenities: DataTypes.STRING,
```

Comment:
This accommodation model doesn't fulfill all required columns and as I can see there no relationships